file_store/src/mobile_session.rs

```diff
@@ -203,14 +203,16 @@ impl TryFrom<DataTransferSessionReqV1> for DataTransferSessionReq {
     }
 }
 
+#[allow(deprecated)]
 impl From<DataTransferSessionReq> for DataTransferSessionReqV1 {
     fn from(v: DataTransferSessionReq) -> Self {
         let report: DataTransferEventProto = v.data_transfer_usage.into();
         Self {
             data_transfer_usage: Some(report),
-            reward_cancelled: v.reward_cancelled,
+            rewardable_bytes: v.rewardable_bytes,
             pub_key: v.pub_key.into(),
             signature: v.signature,
+            reward_cancelled: false,
```

Comment:
wont this would break the ability to replay packets prior to this change ?
You could address this by adding a method to generate rewardable_bytes value for older packets from the existing reward_cancelled/downloaded_bytes/uploaded_bytes fields

Reply:
I could also just continue reading the reward_cancelled field from the proto as longs as we are ok with marking the function `#[allow(deprecated)]`

Reply:
perhaps but a helper method that generates the rewardable_bytes value from a sum of upload/download bytes for those packets which do not include the rewardable_bytes field and which have reward_cancelled: false feels like a better long term fit